rust/macros/module.rs

```diff
@@ -354,8 +354,13 @@ impl ModuleInfo {
                 }
                 "params" => info.params = Some(expect_group(it)),
                 _ => panic!(
-                    "Unknown key \"{}\". Valid keys are: {:?}.",
-                    key, EXPECTED_KEYS
+                    "Unknown key \"{}\". Valid keys are: \"{}\".",
+                    key,
+                    ALL_KEYS
+                        .iter()
+                        .map(|e| e.1)
+                        .collect::<Vec<&'static str>>()
```

Comment:
nit: you don't have to spell out the full type here. Just write `Vec<_>` and compiler will infer that it's a `Vec<&'static str>`.